source/components/inputs/input.ts

```diff
@@ -1,20 +1,24 @@
-import { AfterViewInit, OnInit, EventEmitter } from '@angular/core';
+import { Component, AfterViewInit, OnInit, EventEmitter, AnimationEntryMetadata } from '@angular/core';
 import { FormControl } from '@angular/forms';
 
 import { services } from 'typescript-angular-utilities';
 import __object = services.object;
 import __guid = services.guid;
 
 import { FormComponent } from '../form/form';
+import { slide } from '../../animations/index';
 
-export const baseInputs: string[] = ['name', 'label', 'value', 'disabled'];
+export const baseInputs: string[] = ['name', 'label', 'value', 'disabled', 'labelState', 'hidePlaceholder'];
```

Comment:
I don't think these need to go in the inputs array since these are just private properties to the input class